website/test/CMakeLists.txt

```diff
@@ -0,0 +1,71 @@
+# Copyright (C) 2020  Matthew Glazar
```

Comment:
Blocking: I don't think you meant to add these files in website/. Remove them.

Reply:
Fixed in latest commit:
rebased to latest qljs master branch